server/src/operators/organization_operator.rs

```diff
@@ -187,3 +186,105 @@ pub async fn get_org_from_dataset_id_query(
 
     Ok(organization)
 }
+
+pub fn get_org_dataset_count(
+    organization_id: uuid::Uuid,
+    pool: web::Data<Pool>,
+) -> Result<i64, DefaultError> {
+    use crate::data::schema::datasets::dsl as datasets_columns;
+
+    let mut conn = pool.get().map_err(|_| DefaultError {
+        message: "Could not get database connection",
+    })?;
+
+    let dataset_count = datasets_columns::datasets
+        .filter(datasets_columns::organization_id.eq(organization_id))
+        .count()
+        .get_result(&mut conn)
+        .map_err(|_| DefaultError {
+            message: "Error loading org datasets count",
+        })?;
+
+    Ok(dataset_count)
+}
+
+pub fn get_user_org_count(
+    organization_id: uuid::Uuid,
+    pool: web::Data<Pool>,
+) -> Result<i64, DefaultError> {
+    use crate::data::schema::user_organizations::dsl as user_organizations_columns;
+
+    let mut conn = pool.get().map_err(|_| DefaultError {
+        message: "Could not get database connection",
+    })?;
+
+    let user_count = user_organizations_columns::user_organizations
+        .filter(user_organizations_columns::organization_id.eq(organization_id))
+        .count()
```

Comment:
Instead of doing counts we should have a table which stores the counts like we have for notifs. Gets un performant otherwise